applications/OptimizationApplication/python_scripts/utilities/opt_line_search.py

```diff
@@ -0,0 +1,43 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.OptimizationApplication as KratosOA
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_problem import OptimizationProblem
+from KratosMultiphysics.OptimizationApplication.utilities.component_data_view import ComponentDataView
+
+def CreateLineSearch(parameters: Kratos.Parameters, optimization_problem: OptimizationProblem):
+    type = parameters["type"].GetString()
+    if type == "const_step":
+        return ConstStep(parameters, optimization_problem)
+    else:
+        raise RuntimeError(f"CreateConvergenceCriteria: unsupported convergence type {type}.")
+
+class ConstStep(object):
+    @classmethod
+    def GetDefaultParameters(cls):
+        return Kratos.Parameters("""{
+            "type"              : "const_step",
+            "init_step"          : 0,
+            "gradient_scaling": "inf_norm"
+        }""")
+
+    def __init__(self, parameters: Kratos.Parameters, optimization_problem: OptimizationProblem):
+        parameters.ValidateAndAssignDefaults(self.GetDefaultParameters())
```

Comment:
@Igarizza at this point you need to crosscheck the type of gradient_scaling. According to this implementation whatever I give it does inf_norm scaling. So please check the type. Could you please cover l2_nom as well?! If not I am fine too.

Reply:
added, and no scaling option as well